docs/content/latest/yugabyte-cloud/cloud-security/_index.md

```diff
@@ -26,9 +26,9 @@ Under the shared responsibility model, there is a division of responsibility for
 
 ![Shared responsibility model](/images/yb-cloud/cloud-shared-responsibility.png)
 
-You are responsible for reviewing Yugabyte Cloud security capabilities and features to determine whether your data can be stored in Yugabyte Cloud. You are also responsible for the security, protection, and backup of your data, as well as your use and configuration of Yugabyte Cloud. 
+You are responsible for reviewing Yugabyte Cloud security capabilities and features in accordance with your compliance and regulatory requirements to determine whether your data can be stored in Yugabyte Cloud. You are also responsible for the security, protection, and backup of your data as far as this is possible in the shared responsibility model, as well as for configuring Yugabyte Cloud security and product features to the extent that Yugabyte makes possible. 
 
-Public cloud providers are responsible for protecting the infrastructure that runs all of the cloud services. This infrastructure is composed of the hardware, software, networking, and facilities. Configuration related to these components is generally not available for you to audit or modify in Yugabyte Cloud. You are responsible for configuring Yugabyte Cloud and your clusters, and can audit and remediate any recommendations for those components.
+Yugabyte leverages public cloud provider PAAS and SAAS offerings to deliver the Yugabyte Cloud service. Under the shared responsibility model, the public cloud providers are responsible for providing and protecting the infrastructure (that is, the hardware, software, networking, and facilities) that runs all of the cloud services that Yugabyte provides in Yugabyte Cloud. Configuration related to infrastructure managed and operated by the public cloud providers is generally not available for Yugabyte or Yugabyte customers to audit or modify in Yugabyte Cloud. 
```

Comment:
The second sentence "Under ..." is a bit confusing, with too many "that". Perhaps breaking it into bite-size chunks would help.